### PR TITLE
Frontend build and Unit tests

### PR DESCRIPTION
**Pull request by JonnyEvans99**

### CODE REVIEW

**Review by cormac-larkin-kainos — APPROVED** [review]

File: controller/JobController.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import JobService from "../service/JobService";
+
+const jobService = require('../service/JobService')
+
+module.exports = function(app: Application){
+
+    let jobservice: JobService = new JobService();
+
+    app.get("/jobs", async (req: Request, res: Response) => {
+        console.log("Hello")
```
Comment (shaunganley) [inline controller/JobController.ts:12]: remove these logs

File: model/Job.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export class Job {
+   job_id?: number
+    job_title?: string
```
Comment (shaunganley) [inline model/Job.ts:3]: don't think these need to be optional

File: service/JobService.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+import { Job } from "../model/Job";
+
+export default class JobService {
+
+    URL: string = "http://localhost:8080/api/jobs";
```
Comment (shaunganley) [inline service/JobService.ts:6]: don't hardcode http://localhost:8080/api, set it as an env var and pull it in

File: service/JobService.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+    async getAllJobs(): Promise<Job[]>{
+        try{
+            const response = await axios.get(this.URL);
+            const jobs: Job[] = response.data;
+            console.log("Jobs -> ", jobs);
```
Comment (shaunganley) [inline service/JobService.ts:12]: remove logs

**Review by shaunganley — CHANGES_REQUESTED** [review]

**Comment by shaunganley** [comment]
also, update the readme to run tests

**Review by shaunganley — APPROVED** [review]

**Pull request merged**